react-fm/src/pages/Match-ques/index.ts

```diff
@@ -0,0 +1,3 @@
+import MatchQue from './que';
```

Comment:
```suggestion
import MatchQue from './queue';
```

Spelling check please

Reply:
resolved.